src/js/components/products/ProductsList.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { fetchTranslations } from 'actions';
+import ProductsListFilters from 'components/products/ProductsListFilters';
+import ProductsListHeader from 'components/products/ProductsListHeader';
+import ProductsListTable from 'components/products/ProductsListTable';
+
+
+const ProductsList = (props) => {
+  // Filter params are stored here, to be able to pass them to table component
+  const [filterParams, setFilterParams] = useState({});
+
+  useEffect(() => {
+    props.fetchTranslations(props.locale, 'purchaseOrder');
```

Comment:
wrong translation fetched
change to
`props.fetchTranslations(props.locale, 'productsList');`